sips/sip-16/sip-016-token-metadata.md

```diff
@@ -0,0 +1,394 @@
+      "value": 99,
+    },
+  ],
+  "properties": {
+      "collection":  "Foo Collection",
+      "total_supply":  "10000"
+  },
+  "localization": {
+      "uri": "ipfs://somerandomcid/{locale}.json",
+      "default": "en",
+      "locales": ["en", "pt-BR", "de"]
+  }
+}
+```
+
+de.json
+
+```
+{
+    "sip": 16,
+    "attributes: [
+        {
+          "trait_type": "Haare",
+          "value": "rot",
+        },
+        {
+          "trait_type": "Stärke",
+          "display_type": "number",
+          "value": 99,
+        },
+    ]
+}
+```
+
+pt-BR.json
+
+```
+{
+    "sip": 16,
+    "attributes: [
+        {
+          "trait_type": "cabelos",
+          "value": "vermelho",
+        },
+        {
+          "trait_type": "amido",
+          "display_type": "number",
+          "value": 99,
+        },
+    ]
+}
+```
+
+### Properties
+
+Common properties are described in appendix C. Properties of type object are usually described using the following schema:
+
+```
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Token Metadata Property",
+    "type": "object",
+    "required": [],
+    "properties": {
+        "type": {
+            "type": "string",
+            "description": "type of the property"
+        },
+        "description": {
+            "type": "string",
+            "description": "description of the property"
+        },
+        "value": {
+            "type": {"oneOf": [{"type": "object"}, {"type": "string"}, {"type": "number"}, {"type": "integer"}, {"type": "boolean"} {"type: "array"}},
+            "description": "value of the property"
+        }
+    }
+}
+```
+
+Example:
+
+```
+{
+    "type": "string",
+    "description": "Address of custodian key holder",
+    "value": "Casa Inc., P.O. Box 20575, Charleston, S.C. 29413, United States."
+}
+```
+
+### Attributes
+
+Attributes describe addition elements of tokens that are "observable", usually represented in the image of the token.
+
+In contrast, properties describe elements of tokens that are more abstract.
+
+An attribute consists of a `trait_type` defining the name of the trait, `value` is the value of the trait, and `display_type` is a field indicating how you would like it to be displayed.
+
+Appendix B describes type of attributes
+
+## Localization
+
+The localized data follow the same JSON schema with property `sip` as required and all other properties as optional.
+
+The localized data overwrite data provided in the default meta data JSON. The localized data can provide only partial data.
+
+An array of localized `attributes` overwrites the whole list of default `attributes`.
+
+A localized properties with partial data overwrites only the provided properties, the remaining default properties remain as default values.
+
+# Using metadata in applications
+
+Before presenting metadata to users, application developers should verify whether the metadata is compliant with the application's guidelines.
```

Comment:
Can you speak more to what these application's guidelines are, and where they could be found?  Is there a real-world example you can reference?

Reply:
I added more details.